webpack/assets/javascripts/react_app/components/Layout/Navigation.js

```diff
@@ -71,15 +78,22 @@ const Navigation = ({
             } else {
               groups[currIndex].groupItems.push({
                 ...sub,
-                isActive: currentPath === sub.href?.split('?')[0],
+                isActive:
+                  currentPath === sub.href?.split('?')[0] ||
+                  sub.title === currentLocation ||
+                  (currentLocation === undefined &&
+                    sub.title === 'Any Location') ||
```

Comment:
This wont work if translations is on, so maybe check the classname itself?

Reply:
Thanks for noticing, I changed `(currentLocation === undefined && sub.title === 'Any Location')`, but I think `sub.title === currentLocation` should be alright?